tmk_core/common/action.h

```diff
@@ -88,6 +88,7 @@ void process_record(keyrecord_t *record);
 void process_action(keyrecord_t *record, action_t action);
 void register_code(uint8_t code);
 void unregister_code(uint8_t code);
+inline void tap_code(uint8_t keycode) { register_code(keycode); unregister_code(keycode); }
```

Comment:
Just one more thing: rename `keycode` to `code` so it's consistent with the other two functions. (Sorry I didn't notice this in the previous one.)